javaagent/build.gradle.kts

```diff
@@ -73,6 +78,10 @@ dependencies {
 
   exporterLibs(project(":javaagent-exporters"))
 
+  exporterSlimLibs("io.opentelemetry:opentelemetry-exporter-otlp")
+  exporterSlimLibs("io.opentelemetry:opentelemetry-exporter-otlp-metrics")
+  exporterSlimLibs("io.grpc:grpc-okhttp:1.41.0")
```

Comment:
Just realized that because this is using `okhttp`, adding the HTTP exporters as well should be almost no additional bytecode, so maybe worth it?

Reply:
Well assuming it doesn't cause dependency hell, wow that's old

https://github.com/grpc/grpc-java/blob/master/build.gradle#L158

Reply:
Err - nevermind, yeah it wouldn't cause dependency hell since it's a different artifact

https://github.com/open-telemetry/opentelemetry-java/blob/main/dependencyManagement/build.gradle.kts#L78

So scratch the almost-no-additional-bytecode idea.

Reply:
By the way, just some random trivia https://github.com/grpc/grpc-java/issues/6119#issuecomment-529990399

Reply:
> "OkHttp" name for the transport is a bit of a misnomer

this thread helped my mental picture 👍

Reply:
It appears that we can exclude okhttp 2 from the transitive dependencies

https://github.com/open-telemetry/opentelemetry-java/pull/3678

though I added a comment on the grpc-java issue to see if there is anything obviously bad with that.

Currently I guess we effectively have two copies of okhttp2, the forked-into-grpc one and the transitive dependency which we don't seem to need.

Reply:
I added #4240 to discuss/track